tests/zfs-tests/tests/functional/l2arc/l2arc_arcstats_pos.ksh

```diff
@@ -44,14 +46,14 @@ log_assert "L2ARC MFU/MRU arcstats do not leak."
 
 function cleanup
 {
-	if poolexists $TESTPOOL ; then
-		destroy_pool $TESTPOOL
-	fi
+	rm_devs
 
 	log_must set_tunable32 L2ARC_NOPREFETCH $noprefetch
 }
 log_onexit cleanup
 
+mk_devs
```

Comment:
Can you motivate these changes somehow? Why completely remove and recreate the vdevs for each test?